airbyte-integrations/connectors/source-mysql/src/main/kotlin/io/airbyte/integrations/source/mysql/MysqlJdbcPartitionFactory.kt

```diff
@@ -305,6 +306,20 @@ class MysqlJdbcPartitionFactory(
                         val ba = Base64.getDecoder().decode(stateValue!!)
                         Jsons.valueToTree<BinaryNode>(ba)
                     }
+                    LeafAirbyteSchemaType.TIMESTAMP_WITHOUT_TIMEZONE -> {
```

Comment:
Any other we might want to add?

Reply:
I tested DATE and it works out of box, so no need here.